src/common/configuration.ts

```diff
@@ -0,0 +1,72 @@
+import { ConfigurationChangeEvent, WorkspaceConfiguration } from 'vscode'
+import { ScanEngine } from './constants'
+
+export const CONFIGURATION_IDENTIFIER = 'spectral'
+export const USE_IAC_ENGINE_SETTING = 'scan.engines.useIacEngine'
+export const USE_OSS_ENGINE_SETTING = 'scan.engines.useOssEngine'
+export const USE_SECRET_ENGINE_SETTING = 'scan.engines.useSecretsEngine'
+export const INCLUDE_TAGS_SETTING = 'scan.includeTags'
+
+const prefixIdentifier = (settings: Array<string>) =>
```

Comment:
i think we can remove this method and just add the const `CONFIGURATION_IDENTIFIER` before every setting, it will be more simple
for example: `${CONFIGURATION_IDENTIFIER}.scan.engines.useIacEngine`

Reply:
fixed, https://github.com/SpectralOps/vscode-extension/pull/7/commits/e1fded33e247a3fd2e4a7e247f1b8c1868623af6